index.html

```diff
@@ -12,28 +12,191 @@
 </head>
 
 <body>
-  <header>
-    <nav id="menu-top">
-      <ul class="topnav">
-        <li class="topnav-logo"><a href="#">My Logo</a></li>
-        <li class="topnav-right"><a href="#">Home</a></li>
-        <li class="topnav-right"><a href="#">About</a></li>
-        <li class="topnav-right"><a href="#">Portfolio</a></li>
-        <li class="dropdownIcon"><a href="#">&#9776;</a></li>
-      </ul>
-    </nav>
-  </header>
+  <section class="headersec">
+    <header>
+      <nav id="menu-top">
+        <ul class="topnav">
+          <li class="topnav-logo"><a href="#">My Logo</a></li>
+          <li class="topnav-right"><a href="#">Home</a></li>
+          <li class="topnav-right"><a href="#">About</a></li>
+          <li class="topnav-right"><a href="#">Portfolio</a></li>
+          <li class="dropdownIcon"><a href="#">&#9776;</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="about-flex-container">
+      <div class="name">
+        <h1>I’m Lilian
+          Glad to see you!</h1>
+      </div>
+      <div class="description">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+          and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p>
+      </div>
+      <div class="connect">
+        <a href="#" class="connect">LET’S CONNECT</a>
+      </div>
 
-  <section class="flex-container">
-    <div class="name">
-      <h1>I’m Lilian
-        Glad to see you!</h1>
+      <div class="social-icons">
+        <ul class="social-icons">
+          <li class="social-icon">
+            <a href="https://twitter.com/ifnotlily"> <img src="images/twitter.svg" alt="Lilian's Twitter" /> </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://www.linkedin.com/in/lilian-francis/">
+              <img src="images/Linkedin.svg" alt="Lilian's LinkedIn" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://medium.com/@chatwitlily/">
+              <img src="images/medium.svg" alt="Lilian's Medium" /> </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://angel.co/u/lilian-francis"> <img src="images/angellist.svg" alt="Lilian's AngelList" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://github.com/ifnotlily/">
+              <img src="images/github.svg" alt="Lilian's Github" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+  </section>
+  <section class="section-grid">
```

Comment:
You need to fix the styling for the project cards as the difference shown below
#### Original design
![right2](https://user-images.githubusercontent.com/58376020/125164285-b77f1a80-e191-11eb-8d2c-090a0b1f0abf.png)

#### Your Project
![wrong2](https://user-images.githubusercontent.com/58376020/125164291-c1088280-e191-11eb-95e1-ceecd2894d99.png)

Reply:
Resolved